errors/re-export-all-exports-from-page-is-disallowed.md

```diff
@@ -0,0 +1,9 @@
+# Re-exporting all exports from a page is disallowed
+
+#### Why This Error Occurred
+
+This error occurred when use of `export * from '...'` in Next.js pages to prevent errors.
```

Comment:
Can we be more clear here about why/what errors will be prevented.

Reply:
@lfades Updated !